.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
This will probably break some aspects of VSCode. JSON format does not support comments like in JavaScript.